packages/components/inputs/rich-text-utils/src/rich-text-body/dropdown.tsx

```diff
@@ -37,46 +41,37 @@ type THeadings = {
   label?: string;
 };
 
-const getIsSelected = (props: TDropdownProps, item: TItem) =>
-  !props.isMulti
-    ? item.value === (props.value as string)
-    : (props.value as string[]) &&
-      (props.value as string[]).find(
-        (selectedItem) => selectedItem === item.value
-      );
-
 const Label = styled.div;
 
+type DownshiftRenderProps = {
+  isOpen: boolean;
+  getToggleButtonProps: (options?: GetToggleButtonPropsOptions) => unknown;
+  getItemProps: (options: GetItemPropsOptions<unknown>) => unknown;
+};
+
 const Dropdown = (props: TDropdownProps) => {
   const DropdownItem: FunctionComponent<{
-    value: string | string[];
-    isSelected: unknown;
+    value: string;
+    isSelected: boolean;
   }> = props.components.Item;
   const DropdownLabel = props.components.Label;
-  const isIndeterminate: boolean =
+
+  const isIndeterminate =
     props.isMulti &&
-    (props.value as unknown[]) &&
-    (props.value as unknown[]).length > 0;
+    props.options.some((item) => props.getIsItemSelected(item) === true);
 
   const Downshift = DownshiftUntyped as ElementType;
 
   return (
     <Downshift
       onChange={props.onChange}
-      selectedItem={props.value}
       itemToString={(headings: THeadings) => headings?.label || ''}
     >
       {({
         isOpen,
         getToggleButtonProps,
         getItemProps,
-      }: {
-        isOpen: boolean;
-        getToggleButtonProps: (
-          options?: GetToggleButtonPropsOptions
-        ) => unknown;
-        getItemProps: (options: GetItemPropsOptions<unknown>) => unknown;
-      }) => {
+      }: DownshiftRenderProps) => {
```

Comment:
I think Downshift has its one types. Can you check if we still need to provide our "own" types `DownshiftRenderProps`?